server.go

```diff
@@ -78,7 +96,7 @@ func main() {
 		panic(err)
 	}
 
-	db, _ := sql.Open("mysql", fmt.Sprintf("%s:%s@/", *mysqlUser, *mysqlPassword))
+	db, _ := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s:%d)/", *mysqlUser, *mysqlPassword, *mysqlHost, *mysqlPort))
```

Comment:
Hmm, why are we ignoring the `error` returned in `sql.Open(...)` here? Just curious. Shouldn't you just check for it and `panic(err)` if it occurred?

My .02